### PR TITLE
Adding new intents + training examples

### PR DESCRIPTION
**Pull request by luiseloi**
Adding new intents, namely: online_banking, retirement_plan, change_pin, interest_rate. Also contributing with some extra examples on transfer_money. I'd suggest the duplicate intents to be merged together.

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @samsucik will take a look at it as soon as possible ✨

**Comment by samsucik** [comment]
Hi @luiseloi, thanks a lot for the contribution! Before we go ahead and merge it, could you add the new intents to the README as well? Additionally, I agree that `transfer_money` shouldn't be twice in the NLU file. Do you want to go ahead and merge the two groups of `transfer_money` examples together? :-)

**Comment by luiseloi** [comment]
Hi, sure! Still starting to contribute, so I didn't want to mess up with the work already done there. I also merged the currency duplicates to keep it clean.

File: banking/nlu.md [hunk 1]
```diff
@@ -352,4 +350,66 @@
...
+ - I really don't remember where my card is
+ - I'm afraid I don't know where my card is
+ - I've lost my credit card and I don't think I will find it again
+ - My card is gone I must have left it somewhere
+ - I'm afraid someone took my card so I'd better cancel it
+ - I have no clue where is my card 
+ - Can't find my card don't know what to do
+ - Can't find my card what should I do
+
+# intent:change_pin <!--The user wants to change his/her pin code-->
```
Comment (samsucik) [inline banking/nlu.md:382]:
```suggestion
# intent:change_pin <!--The user wants to change their pin code-->
```

File: banking/nlu.md [hunk 2]
```diff
@@ -352,4 +350,66 @@
...
+ - yeah hi what is the current interest rate for loans
+ - what's the interest rate for a 30 year mortgage
+ - what's the availavble interest rate for mortgages today
+ - what's the interest rate for your golden credit cards 
+ - what's the interest rate for your credit card
+ - What's the best interest rate I can get within your bank
+ - I need to borrow some money so I want to know the interest rates
+ - What are the interest rates if I ask for one hundred thousand
+
+# intent:lost_card <!--The user lost his/her card-->
```
Comment (samsucik) [inline banking/nlu.md:367]:
```suggestion
# intent:lost_card <!--The user lost their card-->
```

**Review by samsucik — CHANGES_REQUESTED** [review]
Looks good, thanks! I've left two minor comments :-)

**Comment by luiseloi** [comment]
Cool, thanks :)

**Review by EmmaWightman — APPROVED** [review]

**Review by EmmaWightman — APPROVED** [review]

**Pull request merged**